Minio/V4Authenticator.cs

```diff
@@ -71,9 +71,10 @@ internal class V4Authenticator : IAuthenticator
         /// <summary>
         /// Authenticator constructor.
         /// </summary>
-        /// <param name="scheme">URL Scheme</param>
+        /// <param name="secure"></param>
         /// <param name="accessKey">Access key id</param>
         /// <param name="secretKey">Secret access key</param>
+        /// <param name="region"></param>
         public V4Authenticator(bool secure,string accessKey, string secretKey, string region ="us-east-1")
```

Comment:
This API has been changed region is not available anymore, we can remove the reference now?